trade-finance-manager-ui/server/controllers/utilisation-reports/helpers/set-record-correction-status.ts

```diff
@@ -1,34 +1,27 @@
-/**
- * returns the record correction status string
- * based on if record correction is completed or not
- * @param isCompleted - if record correction is completed
- * @returns 'Record correction sent' or 'Record correction received'
- */
-export const setRecordCorrectionDisplayStatus = (isCompleted: boolean) => {
-  const recordCorrectionSent = 'Record correction sent';
-  const recordCorrectionReceived = 'Record correction received';
-
-  if (isCompleted) {
-    return recordCorrectionReceived;
-  }
-
-  return recordCorrectionSent;
-};
+import { RecordCorrectionStatus, RecordCorrectionDisplayStatus } from '../../../types/view-models';
 
 /**
- * sets the record correction status based on isCompleted
- * if true, then returns RECORD_CORRECTION_RECEIVED
- * else returns RECORD_CORRECTION_SENT
+ * returns displayStatus and status
+ * based on if record correction is completed or not
  * @param isCompleted - if record correction is completed
- * @returns RECORD_CORRECTION_RECEIVED if completed else RECORD_CORRECTION_SENT
+ * @returns displayStatus and status
  */
 export const setRecordCorrectionStatus = (isCompleted: boolean) => {
-  const recordCorrectionSent = 'RECORD_CORRECTION_SENT';
-  const recordCorrectionReceived = 'RECORD_CORRECTION_RECEIVED';
+  const recordCorrectionDisplayStatusSent: RecordCorrectionDisplayStatus = 'Record correction sent';
+  const recordCorrectionDisplayStatusReceived: RecordCorrectionDisplayStatus = 'Record correction received';
+
+  const recordCorrectionStatusSent: RecordCorrectionStatus = 'RECORD_CORRECTION_SENT';
+  const recordCorrectionStatusReceived: RecordCorrectionStatus = 'RECORD_CORRECTION_RECEIVED';
```

Comment:
I don't mind this pattern personally but I thought this was the sort of thing we are supposed to be introducing constants for? e.g.
```suggestion
  const recordCorrectionDisplayStatusSent: RecordCorrectionDisplayStatus = RECORD_CORRECTION_DISPLAY_STATUS.RECORD_CORRECTION_SENT;
  const recordCorrectionDisplayStatusReceived: RecordCorrectionDisplayStatus = RECORD_CORRECTION_DISPLAY_STATUS.RECORD_CORRECTION_RECEIVED;

  const recordCorrectionStatusSent: RecordCorrectionStatus = RECORD_CORRECTION_STATUS.RECORD_CORRECTION_SENT;
  const recordCorrectionStatusReceived: RecordCorrectionStatus = RECORD_CORRECTION_STATUS.RECORD_CORRECTION_RECEIVED;
```